pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -360,10 +377,58 @@ func (s *Summarizer) save() error {
 	jsonWriter := io.Writer(jsonFile)
 	encoder := json.NewEncoder(jsonWriter)
 	encoder.SetIndent("", " ")
+
+	avgroup := []*ActualValueGroup{}
+
+	for _, gw := range s.fullReport.GroupWrappers {
+		avchecks := []*ActualValueCheck{}
```

Comment:
Let's set the slice capacity, otherwise the runtime will have to do the heavy lifting of expanding the backing array as it grows. 

```suggestion
		avchecks := make([]*ActualValueCheck, 0, len(gw.CheckWrappers))
```

Reply:
done